frontend/views/containers/chatroom/image-viewer/ImageViwerModal.vue

```diff
@@ -0,0 +1,233 @@
+<template lang="pug">
+.c-image-viewer-modal
+  .c-image-viewer-content
+    .c-image-blurry-background(:style='blurryBgStyles')
+    preview-image-area(:img-src='metaData.imgUrl')
+
+    header.c-modal-header
+      avatar-user.c-avatar(
+        v-if='metaData.ownerID'
+        :contractID='metaData.ownerID'
+        size='sm'
+      )
+
+      .c-img-data
+        .c-name.has-ellipsis {{ displayName }}
+        .c-filename.has-ellipsis {{ metaData.name }}
+
+      button.is-icon-small.c-close-btn(
+        type='button'
+        @click.stop='close'
+      )
+        i.icon-times
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import sbp from '@sbp/sbp'
+import trapFocus from '@utils/trapFocus.js'
+import { CLOSE_MODAL } from '@utils/events.js'
+import AvatarUser from '@components/AvatarUser.vue'
+import PreviewImageArea from './PreviewImageArea.vue'
+
+export default {
+  // NOTE: gave this component a generic name in case this is used outside the chatroom area. (eg. instead of 'ChatImageViewer' etc.)
+  name: 'ImageViewerModal',
```

Comment:
Looks like the filename has a typo in it :)

Reply:
good catch. fixed.